### DISCUSSION TITLE
Can't halt rollout

### DISCUSSION THREAD

**Discussion by kiloT** [fastlane/fastlane#21265] [Q&A]
Trying to halt rollout from fastlane. Trying it in all possible variations, but no success.

More closer to success is:
    upload_to_play_store(
      version_code:"10104",
      track: "production",
      track_promote_to:"production",
      track_promote_release_status:"halted",
      skip_upload_apk: true,
      skip_upload_aab: true,
      skip_upload_metadata: true,
      skip_upload_images: true,
      skip_upload_screenshots: true,
      skip_upload_changelogs: true,
    )

has error: 
- _Google Api Error: Invalid request - HALTED release must have fraction_

if i add rollout param(rollout = "0") to action then have error:
- _Invalid value '0', must be greater than 0.0 and less than 1.0_

if i add rollout param(rollout = "0.1" for example) to action then do not have errors but nothing happen(release is still inProgress):

Please, help to halt rollout =)

**Comment by nico-gonzalez** [comment]
@kiloT I'm having the same issue. Do you have any update? were you able to get this working?

**Reply by kiloT** [reply]
Hi) no, did not find any variants to fix it. now i am do halting manually)

**Reply by nico-gonzalez** [reply]
Thanks for the reply. I created this issue https://github.com/fastlane/fastlane/issues/21431
